src/Sylius/Bundle/CoreBundle/Fixture/Factory/OrderExampleFactory.php

```diff
@@ -0,0 +1,239 @@
+
+    /** @var RepositoryInterface */
+    private $channelRepository;
+
+    /** @var RepositoryInterface */
+    private $customerRepository;
+
+    /** @var RepositoryInterface */
+    private $productRepository;
+
+    /** @var RepositoryInterface */
+    private $countryRepository;
+
+    /** @var PaymentMethodRepositoryInterface */
+    private $paymentMethodRepository;
+
+    /** @var ShippingMethodRepositoryInterface */
+    private $shippingMethodRepository;
+
+    /** @var FactoryInterface */
+    private $addressFactory;
+
+    /** @var StateMachineFactoryInterface */
+    private $stateMachineFactory;
+
+    /** @var OrderShippingMethodSelectionRequirementCheckerInterface */
+    private $orderShippingMethodSelectionRequirementChecker;
+
+    /** @var OrderPaymentMethodSelectionRequirementCheckerInterface */
+    private $orderPaymentMethodSelectionRequirementChecker;
+
+    /** @var \Faker\Generator */
+    private $faker;
+
+    public function __construct(
+        FactoryInterface $orderFactory,
+        FactoryInterface $orderItemFactory,
+        OrderItemQuantityModifierInterface $orderItemQuantityModifier,
+        ObjectManager $orderManager,
+        RepositoryInterface $channelRepository,
+        RepositoryInterface $customerRepository,
+        RepositoryInterface $productRepository,
+        RepositoryInterface $countryRepository,
+        PaymentMethodRepositoryInterface $paymentMethodRepository,
+        ShippingMethodRepositoryInterface $shippingMethodRepository,
+        FactoryInterface $addressFactory,
+        StateMachineFactoryInterface $stateMachineFactory,
+        OrderShippingMethodSelectionRequirementCheckerInterface $orderShippingMethodSelectionRequirementChecker,
+        OrderPaymentMethodSelectionRequirementCheckerInterface $orderPaymentMethodSelectionRequirementChecker
+    ) {
+        $this->orderFactory = $orderFactory;
+        $this->orderItemFactory = $orderItemFactory;
+        $this->orderItemQuantityModifier = $orderItemQuantityModifier;
+        $this->orderManager = $orderManager;
+        $this->channelRepository = $channelRepository;
+        $this->customerRepository = $customerRepository;
+        $this->productRepository = $productRepository;
+        $this->countryRepository = $countryRepository;
+        $this->paymentMethodRepository = $paymentMethodRepository;
+        $this->shippingMethodRepository = $shippingMethodRepository;
+        $this->addressFactory = $addressFactory;
+        $this->stateMachineFactory = $stateMachineFactory;
+        $this->orderShippingMethodSelectionRequirementChecker = $orderShippingMethodSelectionRequirementChecker;
+        $this->orderPaymentMethodSelectionRequirementChecker = $orderPaymentMethodSelectionRequirementChecker;
+
+        $this->faker = \Faker\Factory::create();
+    }
+
+    protected function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver
+            ->setDefault('amount', 20)
```

Comment:
This option is not used anywhere. We can set things like `channel`, `customer`, or `country`